src/Core/Services/Implementations/StripeSyncService.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+using Bit.Core.Exceptions;
+
+namespace Bit.Core.Services
+{
+    public class StripeSyncService : IStripeSyncService
+    {
+        private readonly IStripeAdapter _stripeAdapter;
+
+        public StripeSyncService(IStripeAdapter stripeAdapter)
+        {
+            _stripeAdapter = stripeAdapter;
+        }
+
+        public async Task UpdateCustomerEmailAddress(string gatewayCustomerId, string emailAddress)
+        {
+            if (string.IsNullOrWhiteSpace(gatewayCustomerId))
+                throw new InvalidGatewayCustomerIdException();
+
+            if (string.IsNullOrWhiteSpace(emailAddress))
+                throw new InvalidEmailException();
```

Comment:
Missing parenthesis? I didn't find anything in our coding guidelines in regards to parenthesis on single line ifs. In case of a `throw` or a `return` I personally don't mind leaving them off, but I think most people would favor adding them anyway.